### PR TITLE
[TASK] Write a chapter on backend login trouble shooting

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main, 13.4, 12.4

### CODE REVIEW

File: Documentation/Troubleshooting/BackendLogin.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Login page reloads without error
+================================
+
+*   Try deleting all browser cookies and caches or use a different browser /
+    device. There might be an outdated cookie that prevents log-in.
+*   Clear all TYPO3 caches via console command or install tool.
+*   Delete folder :path:`var/cache` to remove possible corrupted cache data.
+*   Check the cookies configuration in TYPO3: Ensure that cookie-related
+    settings (:confval:`sessionTimeout <t3coreapi:globals-typo3-conf-vars-be-sessiontimeout>`,
+    :confval:`cookieDomain <t3coreapi:globals-typo3-conf-vars-be-cookiedomain>`)
```
Comment (froemken) [inline Documentation/Troubleshooting/BackendLogin.rst:25]: Should we mention the SYS cookieDomain here, too, as a fallback, if BE cookieDomain is not set?

File: Documentation/Troubleshooting/BackendLogin.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+code in a third party extension, check if a newer version is available. If you
+do not find one you can try to reach out to the extension author or patch it
+yourself.
+
+..  _troubleshooting-login-access-denied:
+
+Access denied before or after login
+===================================
+
+*   If the line above contains "Access denied." or a similar string that sting will
```
Comment (froemken) [inline Documentation/Troubleshooting/BackendLogin.rst:63]:
```suggestion
*   If the line above contains "Access denied." or a similar string that string will
```

File: Documentation/Troubleshooting/BackendLogin.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Troubleshooting/BackendLogin.rst:25]: It is actually mentioned in the describtion of BE cookieDomain?

Comment (froemken) [reply Documentation/Troubleshooting/BackendLogin.rst:25]: Yes: "Same as [$TYPO3_CONF_VARS['SYS']['cookieDomain']](https://134sa.typo3retter.de/typo3/module/tools/settings#SYS-cookieDomain) but only for BE cookies. If empty, $TYPO3_CONF_VARS['SYS']['cookieDomain'] value will be used."

**Review by froemken — APPROVED** [review]

**Pull request merged**